### ISSUE TITLE
Click event

### ISSUE THREAD

**Issue by ado2000** [FlowingCode/ChipFieldAddon#7]
Hi, is there a way to catch the click event ?

So, when a user click on a chip, we can do something, like show the chip detail or edit the chip data.

Thanks

**Comment by ado2000** [comment]
Hi Javier, 
thanks a lot, is the new 2.2.0 version available on Vaadin portal / Maven ?

Bye

**Comment by javier-godoy** [comment]
Hello. Not yet but it should be released next week (2.2.0 is code frozen, but we are currently discussing some [Commit Message Guidelines](https://github.com/FlowingCode/DevelopmentConventions/blob/conventional-commits-draft/conventional-commits.md) in order to facilitate browsing the commit history, because it's difficult to tell the real changes apart from demo improvements; and we would like to do the release after rewriting the commit history based on those guidelines). 

Meanwhile, you can use 2.2.0-SNAPSHOT from:
```
<repository>
    <id>FlowingCode Snapshots</id>
    <url>https://maven.flowingcode.com/snapshots</url>            
    <snapshots>
        <enabled>true</enabled>
    </snapshots>
    <releases>
        <enabled>false</enabled>
    </releases>
</repository>
```

The snapshot was built on Nov 19th, and as I mentioned above, there will be no more changes to the addon code.
